tests/testthat/_snaps/eval-walk.md

```diff
@@ -168,3 +168,11 @@
       Error in `select_loc()`:
       ! The RHS of `.data$rhs` must be a symbol.
 
+# can forbid empty selection
+
+    Code
+      ensure_named(integer(), allow_empty = FALSE)
+    Condition
+      Error in `check_empty()`:
```

Comment:
I don't understand why it's showing `check_empty()` here.

Reply:
I think because `call` was `missing` previously